### PR TITLE
GraphQL: add more attributes

### PR DESCRIPTION
**Pull request by saerdnaer**
based on work from @barnslig in https://github.com/barnslig/mediacccde-graphql/blob/master/src/schema.graphql

### CODE REVIEW

File: app/graphql/types/conference_type.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def id
       object.acronym
     end
     def lectures
-      object.events
```
Comment (saerdnaer) [inline app/graphql/types/conference_type.rb]: @manno Any idea how to cast the array object.event from `[Events]` to `[Frontend::Event]` – I think becomes does not work for arrays...

Comment (manno) [reply app/graphql/types/conference_type.rb]: No idea, you probably have to do it for all elements: `events.map { |e| e.becomes(class)! }`.

But why do you need that? Maybe there is a method in Frontend that can be moved into the base class easily?

Comment (saerdnaer) [reply app/graphql/types/conference_type.rb]: That would mean that we move about 5-10 methods form the frontend model to the base class – if it is okay for you, I can do that..

Comment (manno) [reply app/graphql/types/conference_type.rb]: Well, frontend was meant for html/display related methods. If those are needed in the backend now.. You deicde.

If it's 10-20 methods, we should consider  a third layer? `Frontend < CoolNewName < Base`?

**Pull request merged**